qrcode_detection_package/main.py

```diff
@@ -10,40 +10,35 @@
 class QRCodeScannerNode(CommonNode):
     def __init__(self, id: str):
         super().__init__(id)
+        self.qr_code_detector = cv2.QRCodeDetector()
         self.qr_publisher = self.create_publisher(
             String, 'qr_codes', 10)
 
-    def capture_image(self):
+    def _capture_image(self):
```

Comment:
Ein Unterstrich bedeutet in Python "protected". "Private" wären zwei Unterstriche.

Reply:
(facepalm)
Da erklärt sich wieder die 2,3 in Python :D